server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -129,49 +130,105 @@ private JSONObject makeInvalidRequestMessage(String reason) {
   private JSONObject makeMoveNotification(Suspect suspect, ILocation location) {
     JSONObject move = new JSONObject();
     move.put(Constants.EVENT_TYPE, EventType.MOVE_NOTIFICATION);
-    move.put(Constants.SUSPECT, suspect.getName());
+    move.put(Constants.SUSPECT, suspect.toString());
     move.put(Constants.LOCATION, location.toString());
     return move;
   }
-
+  
+  private Player findPlayerWithEvidence(CaseFile casefile, Player suggester) {
+	  List<ICard> suspect = new ArrayList<ICard>();
+	  suspect.add(casefile.getRoom());
+	  Player playerWithEvidence = null;
+	  // next Player
+	  int nextPlayerIndex = this.currentTurnIndex + 1;
+	  for (int i = 0; i < players.size(); i++){
+		  Player currentPlayer = players.get((i + nextPlayerIndex) % players.size());
+		  if(currentPlayer.getTag() != suggester.getTag() && 
```

Comment:
You don't need the currentPlayer.getTag() != suggester.getTag() if you limit i to < players.size() - 1.